sleap_nn/config/trainer_config.py

```
@@ -0,0 +1,170 @@
+import attrs
+from omegaconf import OmegaConf
```

Ruff

`omegaconf.OmegaConf` imported but unused

Remove unused import: `omegaconf.OmegaConf`

(F401)